extra-wm/i3-gaps/autobuild/defines

```diff
@@ -8,3 +8,4 @@ BUILDDEP="graphviz doxygen xmlto"
 PKGDES="Improved tiling WM (window manager), with more features"
 
 PKGCONFL="i3"
+PKGPROV="i3"
```

Comment:
I'm not sure how this might work out - did you test that `polybar` could be kept installed even if the standard `i3` is not installed?

Reply:
Yes, tested OK. I will submit a test result screenshot. Moment (de_DE).